internal/admission/validator_test.go

```diff
@@ -293,14 +425,14 @@ func TestKongHTTPValidator_ValidateClusterPlugin(t *testing.T) {
 			}
 			got, got1, err := validator.ValidateClusterPlugin(context.Background(), tt.args.plugin)
 			if (err != nil) != tt.wantErr {
-				t.Errorf("KongHTTPValidator.ValidatePlugin() error = %v, wantErr %v", err, tt.wantErr)
+				t.Errorf("KongHTTPValidator.ValidateClusterPlugin() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
 			if got != tt.wantOK {
-				t.Errorf("KongHTTPValidator.ValidatePlugin() got = %v, want %v", got, tt.wantOK)
+				t.Errorf("KongHTTPValidator.ValidateClusterPlugin() got = %v, want %v", got, tt.wantOK)
 			}
 			if got1 != tt.wantMessage {
-				t.Errorf("KongHTTPValidator.ValidatePlugin() got1 = %v, want %v", got1, tt.wantMessage)
+				t.Errorf("KongHTTPValidator.ValidateClusterPlugin() got message = %v, want %v", got1, tt.wantMessage)
 			}
```

Comment:
Could we convert this to use `assert.Equal` instead?

Reply:
changed to `assert.*`.